File: POO/Constructores_Clases/Class/JAVA/Persona.java
public class Persona {
    private String nombre;
    private int edad;

    // Constructor sin parámetros
    public Persona() {
        this.nombre = "Desconocido";
        this.edad = 0;
    }

    public void mostrar() {
        System.out.println("Nombre: " + nombre + ", Edad: " + edad);
    }

    public static void main(String[] args) {
        Persona persona = new Persona();
        persona.mostrar();
    }
}

File: POO/Constructores_Clases/Class/js/Persona.js
class Persona {
    constructor() { //Constructor sin parámetros
        this.nombre = "Desconocido";
        this.edad = 0;
    }

    mostrar() {
        console.log(`Nombre: ${this.nombre}, Edad: ${this.edad}`);
    }
}

// Uso
const persona = new Persona();
persona.mostrar();